src/http/client/response.cr

```diff
@@ -31,6 +31,10 @@ class HTTP::Client::Response
     @body
   end
 
+  def successful?
```

Comment:
@luislavena great! I had almost the same purpose, but I thought folks here would rather not.

So, lets settle with `response.success?` only?

I'll make the arrangements, remove the merge warning message!